meta-mender-nxp/README.md

```diff
@@ -0,0 +1,47 @@
+# meta-mender-nxp
+
+Mender integration for nxp based boards
+
+The supported and tested boards are:
+
+ - [Olimex iMX8MP-SOM-4GB-IND and iMX8MP-SOM-EVB-IND](https://www.olimex.com/Products/SOM/NXP-iMX8/)
+
+
+Visit the individual board links above for more information on status of the
+integration and more detailed instructions on how to build and use images
+together with Mender for the mentioned boards.
+
+## Dependencies
+
+This layer depends on:
+
+```
+URI: https://github.com/Freescale/meta-freescale-3rdparty
+branch: scarthgap
+revision: HEAD
+```
+
+```
+URI: https://github.com/Freescale/meta-freescale-distro
+branch: scarthgap
+revision: HEAD
+```
+
+```
+URI: https://github.com/boundarydevices/meta-boundary
```

Comment:
Please remove that layer reference.

Reply:
Btw I will add meta-freescale to the README. I've just noticed it is missing there.